zds/member/tests/tests_views.py

```diff
@@ -1730,10 +1730,9 @@ class IpListingsTests(TestCase):
     """Test the member_from_ip function : listing users from a same IPV4/IPV6 address or same IPV6 network."""
 
     def setUp(self) -> None:
-        # self.regular_user = ProfileFactory()
-        # # settings.ZDS_APP["member"]["bot_account"] = self.mas.user.username
-        # self.external = UserFactory(username=settings.ZDS_APP["member"]["external_account"], password="anything")
         self.staff = StaffProfileFactory().user
+        self.regular_user = ProfileFactory()
+        self.regular_user.user.save()
```

Comment:
À moins que tu n'aies constaté un comportement étrange, `ProfileFactory` te donne une instance `Profile` complètement créée, modèles dépendants inclus, donc il est inutile d'appeler `save()` sur l'`User` associé. D'ailleurs, tu ne le fais pas au dessus :) .
```suggestion
        self.regular_user = ProfileFactory()
```

Reply:
C'est rigolo, il dit que tu n'as pas fait de changement sur le code et je ne peux pas le commit automatique...